essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/integrationtest/Sep10Tests.kt

```diff
@@ -30,12 +31,18 @@ class Sep10Tests : AbstractIntegrationTests(TestConfig()) {
           arrayOf(
             CLIENT_WALLET_SECRET,
             CLIENT_WALLET_EXTRA_SIGNER_1_SECRET,
-            CLIENT_WALLET_EXTRA_SIGNER_2_SECRET
-          )
+            CLIENT_WALLET_EXTRA_SIGNER_2_SECRET,
+          ),
         )
     }
   }
 
+  @Test
+  fun testChallengeSerialization() {
```

Comment:
Does this test fail without this change?

Reply:
It doesn't 🤦 . Good catch, let me fix this test.